src/main/java/eu/cessda/pasc/oci/LanguageExtractor.java

```diff
@@ -155,7 +155,6 @@ private CMMStudyOfLanguage getCmmStudyOfLanguage(CMMStudy cmmStudy, String lang,
         Optional.ofNullable(cmmStudy.funding()).map(map -> map.get(lang)).ifPresent(builder::funding);
 
         // #502 - Use any language to set related publications, override with language specific field if present
```

Comment:
I meant to link to this comment so I deleted the first comment and added this. Should be updated to how it works now.

Reply:
Comment removed